planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -85,6 +85,7 @@ public static String generateDam(String adp, String monitorGenURL, String monito
         adpYaml = DamGenerator.manageTemplateMetada(adpYaml);
```

Comment:
I think this would be much easier if we follow the same pattern used by the [matchmaker](https://github.com/SeaCloudsEU/SeaCloudsPlatform/blob/master/planner/matchmaker/src/main/java/eu/seaclouds/planner/matchmaker/Matchmaker.java#L172) - wdyt?